tracer/test/Datadog.Trace.ClrProfiler.IntegrationTests/GraphQLTests.cs

```diff
@@ -79,7 +79,22 @@ public async Task SubmitsTraces(string packageVersion)
         [Trait("Category", "EndToEnd")]
         [Trait("RunOnWindows", "True")]
         public async Task SubmitsTracesWebsockets(string packageVersion)
-            => await RunSubmitsTraces("SubmitsTracesWebsockets", packageVersion, true);
+        {
+            try
+            {
+                // Remove the websockets tests on version 4
+                // Because websockets have some issues on that specific version
+                var ver = new Version(packageVersion);
+                if (ver.Major == 4)
+                {
+                    return;
+                }
+            }
+            catch
+            { }
```

Comment:
I assume the `try-catch` is to handle the "default" empty string `packageVersion`? We tend to use the following pattern to handle that instead:
```suggestion
            // Remove the websockets tests on version 4
            // Because websockets have some issues on that specific version
            // Run tests on GraphQL 5 by default
            if (!string.IsNullOrEmpty(packageVersion) 
              && new Version(packageVersion).Major == 4)
            {
                return;
            }
```